ludwig/config_validation/checks.py

```diff
@@ -477,7 +477,7 @@ def check_llm_finetuning_output_feature_config(config: "ModelConfig"):  # noqa:
     if config.model_type != MODEL_LLM:
         return
 
-    if config.trainer.type != "finetune":
+    if config.trainer.type != "finetune" and config.adapter.pretrained_adapter_weights is not None:
```

Comment:
Should this be an OR? Why does specifying `pretrained_adapter_weights` no longer require that the first output feature be TEXT?

Or is it that we want to make it so that using the none trainer type doesn't require an output feature?

```python
if config.trainer.type == "none":
    return
```

CC: @arnavgarg1